inst/templates/gitignore

```diff
@@ -0,0 +1,61 @@
+# History files
+.Rhistory
+.Rapp.history
+
+# Session Data files
+.RData
+
+# User-specific files
+.Ruserdata
+
+# Example code in package build process
+*-Ex.R
+
+# Output files from R CMD build
+/*.tar.gz
+
+# Output files from R CMD check
+/*.Rcheck/
+
+# RStudio files
+.Rproj.user/
+
+# produced vignettes
+vignettes/*.html
+vignettes/*.pdf
+
+# OAuth2 token, see https://github.com/hadley/httr/releases/tag/v0.3
+.httr-oauth
+
+# knitr and R markdown default cache directories
+*_cache/
+/cache/
+
+# Temporary files created by R markdown
+*.utf8.md
+*.knit.md
+
+# R Environment Variables
+.Renviron
+
+# renv library
+renv/library/
+renv/staging/
+
+# jamovi options js
+jamovi/ui/*.options.js
+
+# packaged module
+*.jmo
+
+# build folders
+build/
+build*/
```

Comment:
oh, you can git rid of the `build*/` rule too ... i was naming the build folders things like `build-R3.6.3-linux` for a short time, but then i moved to `build/R3.6.3-linux` instead. so `build*/` is redundant now.